src/react/components/common/keyboardBinding/keyboardBinding.tsx

```diff
@@ -1,9 +1,10 @@
-import { KeyboardContext, IKeyboardContext } from "../keyboardManager/keyboardManager";
+import { KeyboardContext, IKeyboardContext, KeyEventType } from "../keyboardManager/keyboardManager";
 import React from "react";
 
 export interface IKeyboardBindingProps {
     accelerator: string;
-    onKeyDown: (evt?: KeyboardEvent) => void;
+    keyEventType: KeyEventType;
```

Comment:
Do you think it makes sense to make the event type optional and default to keydown? or just make it required and explicit?  I have not string preference.

Reply:
Yeah, that actually does make sense. I'll make the change